src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LineWrappingHandler.java

```diff
@@ -160,7 +160,8 @@ private static int getFirstNodeIndent(DetailAST node) {
 
         while (curNode != null && curNode != lastNode) {
 
-            if (curNode.getType() == TokenTypes.OBJBLOCK) {
+            if (curNode.getType() == TokenTypes.OBJBLOCK
+                    || curNode.getType() == TokenTypes.SLIST) {
```

Comment:
It's needed because a lambda body is an SLIST, which needs to be captured by the line wrapper.